guides/doc-Content_Management_Guide/topics/Importing_Custom_Content.adoc

```diff
@@ -138,15 +144,20 @@ To create the product, enter the following command:
 ----
 
 [[Importing_Custom_Content-Creating_a_Custom_RPM_Repository]]
-=== Adding a Custom RPM Repository
+=== Adding a {customrpmtitle} Repository
```

Comment:
Upstream this results in "Adding a RPM Repository", I think that the more correct variant here would be "an RPM Repository". Do we have to mind about fixing this or "a RPM" is good enough? This can be fixed by adding an article to the attribute for example.

Reply:
The easiest way around this would be to pluralise it. What do you think? It could get super messy to add the article because in other instances "your" is used for example.

Reply:
Pluralising will do the trick!

Reply:
The same issue occurs with "a OSTree content" vs "a Custom OSTree content"

Reply:
I think it should be OSTree Content without any article.